ultimatejob/settings.py

```diff
@@ -10,6 +10,7 @@
 https://docs.djangoproject.com/en/3.1/ref/settings/
 """
 
+import os
 from pathlib import Path
```

Comment:
see below, you have MERGE conflicts, please rebase w/ upstream

Reply:
Fixed the conflicts :) is it good to go now?